### DISCUSSION TITLE
BetterNether

### DISCUSSION THREAD

**Discussion by Groupix05** [DynamicTreesTeam/DynamicTrees#929] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/betternether

Biggest problem (I think) : Upside down trees

Trees :
- Willow
- (Old Willow)(maybe too big)
- Crimson Glowing
- Sakura
- Anchor
- Rubeus
- Crimson pine
- Big Warped Tree

bushs :
- Sakura

Mushrooms:
- Wart

Cactus :
- Nether Cactus
- Large Red Mushroom
- Large Brown Mushroom

Things idk if it can be dynamic or not:
- Mushroom Fir
- Giant Mold
- Stalagnate

**Comment by swagepic** [comment]
yes i need this

**Comment by Stargaze001** [comment]
This would make the mod more immersive. I hope the devs see this 👍.

**Comment by hartbrody-spec** [comment]
can work with Forge now.
https://www.curseforge.com/minecraft/mc-mods/betternether-forge

**Reply by Stargaze001** [reply]
> can work with Forge now. https://www.curseforge.com/minecraft/mc-mods/betternether-forge

No way!! Thank you!!

**Reply by hartbrody-spec** [reply]
yeah, i was hoping for something like this because every mod in my modpack has Dynamic Trees compat sans betternether. but it seems more realistic now that betternether is actually on a very popular version of Forge. very well-done and well-maintained asw
<img width="2560" height="1440" alt="2026-01-17_00-35-50-png-png" src="https://github.com/user-attachments/assets/5036e754-f593-4709-82ff-bd2656722d43" />

**Reply by Stargaze001** [reply]
> yeah, i was hoping for something like this because every mod in my modpack has Dynamic Trees compat sans betternether. but it seems more realistic now that betternether is actually on a very popular version of Forge. very well-done and well-maintained asw <img alt="2026-01-17_00-35-50-png-png" width="2000" height="1440" src="https://private-user-images.githubusercontent.com/261629702/550220841-5036e754-f593-4709-82ff-bd2656722d43.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzEyMDAyMjEsIm5iZiI6MTc3MTE5OTkyMSwicGF0aCI6Ii8yNjE2Mjk3MDIvNTUwMjIwODQxLTUwMzZlNzU0LWY1OTMtNDcwOS04MmZmLWJkMjY1NjcyMmQ0My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDIxNVQyMzU4NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZWNjMmNiZWRlMDY4MGE2ZGM1NTdjOGVhNTI4MTAwZmJiNWM4ZmQ3MDJhOGE1OWMzYWYwZTYxYmE0Mjk1MmIwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.mdeCzNkUABThOnFg9Rs87_3Duqk5sjSFlu5xPvZz0Ls">

I use Sinytra connector so I really don’t need to worry

**Comment by hartbrody-spec** [comment]
https://www.curseforge.com/minecraft/mc-mods/betternether-neoforge
need not forget Neo

**Reply by exocyt0sis** [reply]
I second that - with Create: Aeronautics released, a ton of Forge 1.20.1 users have embraced NeoForge 1.21.1.

**Comment by dannykim2011** [comment]
I am working on it.

**Comment by dannykim2011** [comment]
https://www.curseforge.com/minecraft/mc-mods/dynamictrees-betternether
https://modrinth.com/mod/dynamictrees-betternether

**Reply by exocyt0sis** [reply]
Excellent work, @dannykim2011 - keep it up.

**Reply by dannykim2011** [reply]
If you install dt plus for 1.21.1 and 1.20.1, you can use 6 types of mushrooms and 2 types of cacti when using all two addons.

**Reply by dannykim2011** [reply]
26.1.2 Porting has been completed, and bug fixes are currently underway. It is expected to be released within 12 hours.

**Comment by dannykim2011** [comment]
Currently, we created add-ons for the famous better-nether betterend mode in the 1.20.1, 1.21.1, and 26.1.2 severs, and the rework of the end umbrella tree has been completed. Hotfix is expected to be released within a week.

**Reply by exocyt0sis** [reply]
Great! Please keep it up - BetterNether is such an improvement to the vanilla implementation, so having it as part of the Dynamic Trees family is a really neat!

**Comment by dannykim2011** [comment]
2.0.0h with bug fixes released
